apps/web/cypress/tests/notification-editor/main-functionality.spec.ts

```diff
@@ -85,10 +88,34 @@ describe('Workflow Editor - Main Functionality', function () {
 
     editChannel('inApp', true);
 
+    cy.getByTestId('notification-template-submit-btn').should('be.disabled');
     cy.getByTestId('feed-button-1-checked');
     cy.getByTestId('create-feed-input').type('test4');
     cy.getByTestId('add-feed-button').click();
     cy.getByTestId('feed-button-2-checked');
+    cy.getByTestId('notification-template-submit-btn').should('not.be.disabled');
+  });
+
+  it('should edit email notification', function () {
+    const template = this.session.templates[0];
+
+    cy.visit('/templates/edit/' + template._id);
+    cy.waitForNetworkIdle(500);
+
+    // edit email step
+    cy.getByTestId('notification-template-submit-btn').should('be.disabled');
+    clickWorkflow();
+    editChannel('email');
+    cy.getByTestId('notification-template-submit-btn').should('be.disabled');
+
+    // edit email editor content
+    cy.getByTestId('email-editor').getByTestId('editor-row').first().click().type('{selectall}{backspace}Hello world!');
```

Comment:
What's this bracketed string `{selectall}`? Just curious.

Reply:
the `{selectall}` is a cypress magic sequence that will select all the input text and with `{backspace}` it will then remove it ;) ref: https://docs.cypress.io/api/commands/type